submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts

```diff
@@ -114,4 +121,28 @@ export class ExperimentHomeComponent implements OnInit {
   onInitModal(obj) {
     this.form.initModal(obj.initMode, obj.initFormType, obj.id, obj.spec);
   }
+
+  getTensorboardInfo(period: number, due: number) {
```

Comment:
it would be better to use `timeout` instead of `due`

Reply:
The limit time is named `due` in timeout() function in rxjs as the following:
`timeout<T>(due: number | Date, scheduler: SchedulerLike = async): MonoTypeOperatorFunction<T>`
So I think it kind of makes sense.

Reply:
Okay, got it.